nucleus/dataset_item.py

```diff
@@ -89,6 +114,58 @@ class DatasetItemType(Enum):
+
+    Note: for 3D data, please include a :class:`.CameraParams` object under a key named
+    "camera_params" within the metadata dictionary. This will allow for projecting
+    3D annotations to any image within a scene.
+
+
+    Attributes:
+        image_location: (required if pointcloud_location not present) The location
+            containing the image for the given row of data. This can be a local path, or a remote URL.
+            Remote formats supported include any URL (http:// or https://) or URIs for AWS S3, Azure, or GCS,
+            (i.e. s3://, gcs://)
+        reference_id: (required) A user-specified identifier to reference the item.
+        metadata: Extra information about the particular dataset item. ints, floats,
+          string values will be made searchable in the query bar by the key in this dict
+          For example, {"animal": "dog"} will become searchable via
+          metadata.animal = "dog"
+
+          Categorical data can be passed as a string and will be treated categorically
+          by Nucleus if there are less than 250 unique values in the dataset. This means
+          histograms of values in the "Insights" section and autocomplete
+          within the query bar.
+
+          Numerical metadata will generate histograms in the "Insights" section, allow
+          for sorting the results of any query, and can be used with the modulo operator
+          For example: metadata.frame_number % 5 = 0
+
+          All other types of metadata will be visible from the dataset item detail view.
+
+          It is important that string and numerical metadata fields are consistent - if
+          a metadata field has a string value, then all metadata fields with the same
+          key should also have string values, and vice versa for numerical metadata.
+          If conflicting types are found, Nucleus will return an error during upload!
+
+          The recommended way of adding or updating existing metadata is to re-run the
+          ingestion (dataset.append) with update=True, which will replace any existing
+          metadata with whatever your new ingestion run uses. This will delete any
+          metadata keys that are not present in the new ingestion run. We have a cache
+          based on image_location that will skip the need for a re-upload of the images,
+          so your second ingestion will be faster than your first.
+          TODOC(Shorten this once we have a guide migrated for metadata, or maybe link
+          from other places to here.)
+        pointcloud_location: (required if image_location not passed) The remote URL
+          containing the pointcloud JSON. Remote formats supported include any URL
+          (http:// or https://) or URIs for AWS S3, Azure, or GCS, (i.e. s3://, gcs://)
+        upload_to_scale: Set this to false in order to use `privacy mode<https://dashboard.scale.com/nucleus/docs/api#privacy-mode>`_. TODOC (update this once guide is migrated).
```

Comment:
I think reST expects the space before the angled bracket. The space isn't actually rendered in the final hyperlink, so nw there.
```suggestion
        upload_to_scale: Set this to false in order to use `privacy mode <https://dashboard.scale.com/nucleus/docs/api#privacy-mode>`_. TODOC (update this once guide is migrated).
```